automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeDataprocSpec.scala

```diff
@@ -96,57 +96,6 @@ class RuntimeDataprocSpec
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
-  "should create a Dataproc cluster with workers and preemptible workers" taggedAs Retryable in { project =>
```

Comment:
this is already covered in `should stop/start a Dataproc cluster with workers and preemptible workers` test below